vmctl-init-conf

```diff
@@ -4,48 +4,45 @@
 # Run source vmctl-init-conf <confdir>
 
 usage() {
-	echo "Usage: source $0 <confdir>"
+  echo "Usage: source $0 <confdir>"
 }
 
 if [ -z "$1" ]; then
-	usage
-	return 2
+  usage
+  return 2
 fi
 
 if [ "$1" = '--help' ] || [ "$1" = '-h' ]; then
-	usage
-	return 2
+  usage
+  return 2
 fi
 
 CONFDIR="$1"
 if [ -n "${BASH_SOURCE[0]}" ]; then
-	VMCTLDIR=$(dirname "$(realpath "${BASH_SOURCE[0]}")")
+  VMCTLDIR=$(dirname "$(realpath "${BASH_SOURCE[0]}")")
 else
-	VMCTLDIR=$(dirname "$(realpath "$0")")
+  VMCTLDIR=$(dirname "$(realpath "$0")")
 fi
 EXAMPLESDIR=$VMCTLDIR/examples/vm
 
 # Create CONFDIR if necessary
 if [ ! -d "$CONFDIR" ]; then
-	mkdir --parent "$CONFDIR"
-	if [ $? -ne 0 ]; then
-		return 1
-	fi
-	cp --recursive --verbose "$EXAMPLESDIR"/* "$CONFDIR"
-	if [ $? -ne 0 ]; then
-		return 1
-	fi
+  if mkdir --parent "$CONFDIR"; then
```

Comment:
This is actually incorrect; it needs to be ! mkdir .... or else it always goes into the return 1